contracts/VoteLockerCurve.sol

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: MIT
 
 // Inspired by the Curve.fi VotingEscrow contract and the mStable Solidity fork.
-//    - Adds compatability with the OpenZeppelin governance stack (4.5.0)
+//    - Adds compatibility with the OpenZeppelin governance stack (4.5.0)
 //    - Removes the separate lockup amount and lockup duration extension functions info
-//      favour of a single function for create or update.
```

Comment:
Discrimination 💀

Reply:
There is a whole bunch of these words that the US decided to change, just to be different. Join team British English though, we have better tea.

Reply:
Joining team British English then. For the tea and classiness